packages/0x.js/src/utils/decorators.ts

```diff
@@ -1,17 +1,37 @@
 import * as _ from 'lodash';
 
-import {AsyncMethod, ZeroExError} from '../types';
+import {AsyncMethod, SyncMethod, ZeroExError} from '../types';
 
 import {constants} from './constants';
 
-export const decorators = {
-    /**
-     * Source: https://stackoverflow.com/a/29837695/3546986
-     */
-    contractCallErrorHandler(target: object,
-                             key: string|symbol,
-                             descriptor: TypedPropertyDescriptor<AsyncMethod>,
-    ): TypedPropertyDescriptor<AsyncMethod> {
+type ErrorTransformer = (err: Error) => Error;
+
+const contractCallErrorTransformer = (error: Error) => {
+    if (_.includes(error.message, constants.INVALID_JUMP_PATTERN)) {
+        return new Error(ZeroExError.InvalidJump);
```

Comment:
Should we be throwing this? Just in case this transformer is used in another chain in the future. It just so happens that it's the last transfer right now.